engine/src/main/java/org/hibernate/validator/internal/engine/path/NodeImpl.java

```diff
@@ -78,6 +80,7 @@ public static NodeImpl createPropertyNode(String name, NodeImpl parent) {
 				null,
 				ElementKind.PROPERTY,
 				EMPTY_CLASS_ARRAY,
+				null,
```

Comment:
gets a bit nasty with all this parameters which gets set to 'null' depending on the use-case. Not sure how one could improve that!?